app/modules/loan/endpoints_loan.py

```diff
@@ -599,6 +614,43 @@ async def create_loan(
                 quantity=itemret.quantity,
             ),
         )
+
+    try:
+        now = datetime.now(UTC)
+        message = Message(
+            context=f"loan-new-{loan.id}-begin-notif",
+            is_visible=True,
+            title="📦 Nouveau prêt",
+            content=f"Un prêt a été enregistré pour l'association {loan.loaner.name}",
+            expire_on=now.replace(day=now.day + 3),
+        )
+        await notification_tool.send_notification_to_user(
+            user_id=loan.borrower_id,
+            message=message,
+        )
+    except Exception as error:
+        hyperion_error_logger.error(
+            f"Error while sending notification to borrower of a new loan, {error}",
+        )
+
+    try:
+        now = datetime.now(UTC)
+        message = Message(
+            context=f"loan-new-{loan.id}-end-notif",
+            is_visible=True,
+            title="📦 Prêt arrivé à échéance",
```

Comment:
What happen if the loan is returned before the due date, or if the date is modified?